lib/model/query/forms.js

```diff
@@ -172,6 +172,11 @@ const del = (form) => ({ run, Assignments }) =>
   Promise.all([ run(markDeleted(form)), Assignments.revokeByActeeId(form.acteeId) ]);
 del.audit = (form) => (log) => log('form.delete', form);
 
+const restore = (form) => ({ run, Forms }) =>
+  run(markUndeleted(form));
+restore.audit = (form) => (log) => log('form.restore', form);
```

Comment:
add form.restore to https://github.com/getodk/central-backend/blob/master/lib/model/query/audits.js#L45-L46 to show up on the frontend